examples/Image-Segmentation.ipynb

```diff
@@ -0,0 +1,342 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Welcome to this tutorial! Have you done with the image segmentation  with crowdsourcing and now you need to somehow get the final results? Find out how to aggregate the data with methods from Crowd-Kit library by Toloka!"
```

Comment:
I've added a couple of comments for Text Aggregation example PR. Please fix ones that apply here too.